ArjunaJTA/jta/pom.xml

```diff
@@ -274,7 +276,7 @@
         <activeByDefault>false</activeByDefault>
         <property>
           <name>arqProfileActivated</name>
-          <value>true</value>
+          <value>arq</value>
```

Comment:
It wouldn't be appropriate to do that in this particular profile. This profile is for enabling arquillian and is not specific to IPv6

It might be appropriate somehow in the arqIPv6 profile proposed for addition below but I think we should then look to be updating other existing profiles in a similar way (if it's possible) such as https://github.com/jbosstm/narayana/blob/445315df26d218ccd78b10cc2bbb70c793959dc4/ArjunaJTA/cdi/pom.xml#L189-L222

Reply:
The IPv6 profile later in the pom activates based on the same property (so arq could be activated on a combination of `arq` and `java.net.preferIPv6Addresses`) so it confusing. Is this a kludge to work around some issue or is there a design behind it - what existing behaviour was broken or what new functionality has exposed the issue?

Reply:
We should add ipv6 testing if we can, that's the motivation.

The implementation is then to try to propose something which resembles https://github.com/jbosstm/narayana/blob/master/ArjunaJTA/cdi/pom.xml but it is further complicated because of the jdk17 profile being needed for CMRIntegrationTest in the jta pom.xml (cdi does not need add-opens/JDK17 profile)